yt/visualization/_colormap_data.py

```diff
@@ -8440,3 +7470,16 @@
 for k,v in color_map_luts.items():
     temp[k+"_r"] = (v[0][::-1], v[1][::-1], v[2][::-1], v[3][::-1])
 color_map_luts.update(temp)
+
+# YTEP-0040 backward compatibility layer
```

Comment:
Note to reviewers: I'm highlighting this since I seems that the web viewer is making it possibly harder to discover.